tests/test_losca_city_planning.py

```
@@ -0,0 +1,90 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.constants import NOT_CLASSIFIED
```

Ruff

`city_scrapers_core.constants.NOT_CLASSIFIED` imported but unused

Remove unused import: `city_scrapers_core.constants.NOT_CLASSIFIED`

(F401)